packages/ethereum-storage/test/ethereum-metadata-cache.test.ts

```diff
@@ -0,0 +1,63 @@
+import { StorageTypes } from '@requestnetwork/types';
```

Comment:
file renamed ! sorry 😬 

the main change is that I dropped 2 tests that were either useless with these changes, or redundant